manual/src/htmlman/.gitignore

```diff
@@ -9,4 +9,4 @@ manual.css
 odoc.css
 highlight.pack.js
 *.htoc
-*.svg
+/*.svg
```

Comment:
I don't understand where this change comes from and what its purpose is -- it seems slightly inconsistent with the other lines in the file, such as the `*.html` line above.

Reply:
This one (and the similar change in manual/src/html_processing/.gitignore) come from an earlier version of #11346 (see the commit sequence from [0a48b2f](https://github.com/ocaml/ocaml/commits/0a48b2fe9e18d733323c2bad7da7bff9cb9583a3)). That PR got simplified prior to merging by @shindere's #11617, but I was nervous/lazy to backport that entire change because it touches lots of things.

If a pattern in a `.gitignore` file begins with a `/`, then it is interpreted relative to the `.gitignore` file (I personally find this very confusing, and indeed got it wrong to start with - `./` seems a much more intuitive prefix for this) - see [gitignore](https://git-scm.com/docs/gitignore#_pattern_format). Here, it corrects the pattern so that `manual/src/htmlman/*.svg` _is_ ignored (these files are cleaned by `Makefile` in the directory above) but stops it from marking the _committed_ svg file in `manual/src/htmlman/fonts` as being ignored. The reason it's needed is because #11346 adds a CI check to ensure that a file marked as "ignored" has not been committed to the repository in error.

Reply:
Thanks for the explanation.

For the record, I don't like this way to do this too much, because I find the per-directory convention implicit and fragile. For a given filetype that contains a mix of sources and build outputs, I would rather either:
1. Explicitly list all files that we know are build outputs, if we have control over this. (Maybe different hevea version generate different sets and such.)
2. ... or explicitly list all files that we know are sources, using negation patterns `!foo` after the catch-all on the extension.

This being said, this is all not-so-relevant for the present PR, so okay.